src/components/atoms.jsx

```diff
@@ -115,13 +198,65 @@ export const External = styled.a`
     color: grey;
   }
 `
+export const SmallExternal = styled.a`
```

Comment:
We're going to want to abstract all of these into a single component that accepts a `size` attribute.